### PR TITLE
Update Developer Guide with the proposed Better CCA-level Actions Meta-feature

### PR DESCRIPTION
**Pull request by JuliaPoo**
Also fixed DG styling.

### CODE REVIEW

**Review by chuahjiajie — COMMENTED** [review]
Looks good to me

**Review by chuahjiajie — APPROVED** [review]
Looks good to me

**Review by chuahjiajie — COMMENTED** [review]
Settled the merge conflict between your part and mine

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -335,6 +336,33 @@ Following the implementation of this meta-feature, the following features can be
...
+2. Increment the attendance value by 1.
+3. Update the attendance details of the person in the model.
+
+An example command for incrementing attendance is ``incatt n\John Doe``. Which means
+"Increment the attendance of John Doe by 1".  The process is as follows.
+
+1. Retrieve the attendance information of John Doe.
+2. Edit the session detail according to the input.
+3. Update the attendance details of the person in the model.
+
```
Comment (JuliaPoo) [inline docs/DeveloperGuide.md:365]: Yay awesome

**Pull request merged**